packages/block-editor/src/components/block-list/block.js

```diff
@@ -215,22 +218,39 @@ function BlockListBlock( {
 		);
 	}
 
-	const value = {
-		clientId,
-		isSelected,
-		isFirstMultiSelected,
-		isLastMultiSelected,
-		isPartOfMultiSelection,
-		enableAnimation,
-		index,
-		className: wrapperClassName,
-		isLocked,
-		name,
-		mode,
-		blockTitle: blockType.title,
-		wrapperProps: omit( wrapperProps, [ 'data-align' ] ),
-	};
-	const memoizedValue = useMemo( () => value, Object.values( value ) );
+	const wrapperPropsOmitAlign = omit( wrapperProps, [ 'data-align' ] );
```

Comment:
Won't this always return a new array?

Reply:
Oops, you're right!

Reply:
Just found out that doesn't that mean the original way of doing this also will create a new array for each render?

Reply:
Yes, you're right. It seems fine to just pass the each wrapper props and include the data attribute in that case.